modules/lib/lib-project/src/main/resources/lib/xp/project.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Functions to manage Content Projects.
+ *
+ * @example
+ * var projectLib = require('/lib/xp/project');
+ *
+ * @module project
+ */
+
+/* global __ */
+
+function required(params, name) {
+    var value = params[name];
+    if (value === undefined) {
+        throw 'Parameter \'' + name + '\' is required';
+    }
+
+    return value;
+}
+
+function nullOrValue(value) {
+    if (value === undefined) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5502328)